animegrep.sh

```diff
@@ -17,43 +17,43 @@
 # Make sure the script fails if the other commands fail as well
 set -eo pipefail
 
-# check for mkvextract
-command -v mkvextract >/dev/null 2>&1 || { echo >&2 "Requires mkvextract. Aborting."; exit 3; }
+print_help() {
+	echo "$0 [-d <directory> | -f <file> ] -t <track> -w <WORD> -m -h "
+	echo ""
+	echo "animegrep is a bash script that extracts the subtitles from an mkv video file, greps them for a specified word, parses them and extracts only that time frame from the source video file"
+	echo ""
+	echo "-d <directory> sets a directory as your source media"
+	echo "-f <file> sets a file as your source media. You cannot set directory and files at the same time."
+	echo "-t <track> Not Documented. "
+	echo "-w <word> track number for your source media."
+	echo "-m to merge multiple output files as one. Useful if you have -d set."
+}
 
-# TODO: convert this to getopts so we don't have to set it the hackiest way possible.
 POSITIONAL=();
```

Comment:
Not POSIX-compatible.

Reply:
POSIX compatibility is out of scope for this PR. I'll make a different branch for that.